db/migrations/00033_group_request_types.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE group_membership_requests ADD COLUMN IF NOT EXISTS kind STRING NOT NULL DEFAULT 'new_member';
+ALTER TABLE group_memberships ADD COLUMN IF NOT EXISTS admin_expires_at TIMESTAMPTZ NULL;
+ALTER TABLE group_membership_requests ADD COLUMN IF NOT EXISTS admin_expires_at TIMESTAMPTZ NULL;
```

Comment:
its interesting that we both duplicate the expiration values and also that we need separate ones for admin and membership - this makes sense (I think) but I didnt think about that prior